### DISCUSSION TITLE
Cannot modify "myusername".github.io

### DISCUSSION THREAD

**Discussion by yuzhenmao** [community/community#21642] [Other Feature Feedback, Questions, & Ideas]
<p>Hi, I created a repo named “myusername”.github.io 5 months ago, and used Jekyll as my template. It works well, I could see my webpage via: “myusername”.github.io. Then I deleted this repo, but the webpage still there. Yesterday, I create a repo named “myusername”.github.io again, and used a different template, but the webpage didn’t change, it was still the old version. I tried every method, for example, set future: true, or modified the timezone, all failed. The old repo seems always there and I couldn’t use my new repo to modify the webpage. Could anyone help me? Thanks!</p>
<hr>
<p>Update: It shows the right webpage after I switch my browser from Chrome to ie. So weired.</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>HI <a class="mention" href="/yuzhenmao">@yuzhenmao</a> and welcome to the community!</p>
<p>Have you tried clearing your browser history in Chrome? There may be cached data.</p>

**Comment by yuzhenmao** [comment]
<p>Thanks, ernest! <a class="mention" href="/ernest-phillips">@ernest-phillips</a> Now it works! :grinning:</p>
